modin/logging/logger_decorator.py

```diff
@@ -79,10 +79,10 @@ def enable_logging(
     log_level = log_level.lower()
     assert hasattr(Logger, log_level.lower()), f"Invalid log level: {log_level}"
 
-    def decorator(obj):
+    def decorator(obj: Any) -> Any:
         """Decorate function or class to add logs to Modin API function(s)."""
         if isinstance(obj, type):
-            seen = {}
+            seen: Dict[Any, Any] = {}
```

Comment:
```suggestion
            seen: Dict[str, Any] = {}
```

Is this correct?

Reply:
I don't think so. If you see the usage of `seen` it seems to be looking things up with `attr_value` as the key, and the type for that seems to be `Any`.

Reply:
It should really be `Dict[Callable, Callable]`, as it's mapping unwrapped function to wrapped function, to handle cases like
```python
@enable_logging
class Foo:
    def bar(self):
        pass
    qux = bar

assert Foo.bar == Foo.qux
```

Reply:
I had tried that before but mypy complained, so I kind of copped out to use the `Any`:
```
modin/logging/logger_decorator.py:91:40: error: Invalid index type "Union[FunctionType, MethodType, classmethod[Any], staticmethod[Any]]" for "Dict[Callable[..., Any], Callable[..., Any]]"; expected type "Callable[..., Any]"  [index]
modin/logging/logger_decorator.py:93:40: error: Invalid index type "Union[FunctionType, MethodType, classmethod[Any], staticmethod[Any]]" for "Dict[Callable[..., Any], Callable[..., Any]]"; expected type "Callable[..., Any]"  [index]
```